### PR TITLE
 LOK-2234 : BE: BFF: Node: DownloadIpInterfacesToCsv

### PR DESCRIPTION
**Pull request by Junaid-Khan-Nant**
## Description
Download IpInterfaces for a given node to CSV. This can be done completely in BFF ( rest-server).

## Jira link(s)
https://opennms.atlassian.net/browse/LOK-2234

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: rest-server/src/main/java/org/opennms/horizon/server/service/GrpcNodeService.java [hunk 1]
```diff
@@ -231,4 +232,48 @@ public Flux<IpInterface> searchIpInterfacesByNodeAndSearchTerm(
                         .map(ipInterfaceMapper::protoToIpInterface)
                         .toList());
     }
+
+    @GraphQLQuery(name = "downloadSearchIpInterfacesByNodeAndSearchTerm")
+    public Mono<IpInterfaceResponse> downloadSearchIpInterfacesByNodeAndSearchTerm(
```
Comment (smunir-onms) [inline rest-server/src/main/java/org/opennms/horizon/server/service/GrpcNodeService.java:237]: It should be "downloadIpInterfaces" OR "searchAndDownloadIpInterfaces" OR "downloadIpInterfacesByNodeAndSearchTerm"

**Review by smunir-onms — COMMENTED** [review]
LGTM, discussed changes verified.

**Review by cgorantla — APPROVED** [review]
LGTM!

**Comment by Junaid-Khan-Nant** [comment]
Merge into develop LOK-2234.

**Pull request merged**